src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --header-color-active: #00acdc;
+  --height-header: 60px;
+  --margin-header: 50px;
+  --bg-color-header: #fff;
```

Comment:
good practice to declare variables inside the [:root](https://www.w3schools.com/css/css3_variables.asp) selector
```css
:root {
  --header-color-active: #00acdc;
  --height-header: 60px;
  --margin-header: 50px;
  --bg-color-header: #fff;
}
```